Server/Plugins/APIDump/APIDesc.lua

```diff
@@ -1730,6 +1730,178 @@ end
 				},
 			},
 		},
+		cColor =
+		{
+			Desc = [[
+				Encapsulates a RGB color, e.g. for armor.
+			]],
+			Functions =
+			{
+				Clear =
+				{
+					Notes = "Resets the color to uninitialized."
+				},
+				constructor =
+				{
+					{
+						Returns = { {Type="cColor"} },
+						Notes = "Creates an uninitialized cColor. Each component must be between 0 and 255, inclusive.",
```

Comment:
`uninitialized`? Since it accepts the RGB components, I'd say it's perfectly initialized.